instrumentation/opentelemetry-instrumentation-wsgi/src/opentelemetry/instrumentation/wsgi/__init__.py

```diff
@@ -304,6 +326,14 @@ def collect_custom_response_headers_attributes(response_headers):
     return attributes
 
 
+def _parse_status_code(resp_status):
+    status_code, _ = resp_status.split(" ", 1)
```

Comment:
Status code [is always 3 digits](https://datatracker.ietf.org/doc/html/rfc2616.html#page-39:~:text=The%20Status%2DCode%20element%20is%20a%203%2Ddigit%20integer%20result%20code%20of%20the), so `status_code[:3]` should work?

Reply:
Yes, that should work too